packages/blocks-ui/src/panels/property-controls.js

```diff
@@ -75,6 +77,42 @@ export default ({
                   ))}
                 </Field>
               )
+            } else if (value.type === ControlType.Boolean) {
+              return (
+                <div key={key}>
+                  <Label>{title}</Label>
+                  <Box
+                    sx={{
```

Comment:
Not sure about your styling convention. Tell me if you don't like the inline ones.